build.rs

```diff
@@ -60,7 +60,12 @@ pub fn find_executable(name: &str) -> Option<PathBuf> {
     let cmd = Command::new(WHICH).arg(name).output().ok()?;
     if cmd.status.success() {
         let stdout = String::from_utf8_lossy(&cmd.stdout);
-        Some(stdout.trim().into())
+        stdout
+            .trim()
+            .replace("\r", "")
+            .split("\n")
```

Comment:
[`.lines`](https://doc.rust-lang.org/std/primitive.str.html#method.lines)?

Also trimming the final line instead of the entire string might be a good idea.